### PR TITLE
AEA-949: New UK Core version. Removing schema from OpenAPI spec.

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
-
 [tool.poetry]
-name = "apim-definitions"
+name = "electronic-prescription-service-api"
 version = "0.0.1-alpha"
 description = "Electronic Prescription Service API Definitions"
-
 license = "MIT"
-
 authors = [
```
Comment (dan-jones-iw) [inline pyproject.toml:9]: alphabetical ordering or order of importance??

Comment (mfjarvis) [reply pyproject.toml:9]: Same ordering as https://python-poetry.org/docs/pyproject/, so I kept it

**Review by dan-jones-iw — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -150,7 +150,7 @@ export APIGEE_ENVIRONMENT=$(env)
 export APIGEE_ACCESS_TOKEN=$(token)
 
 space :=
-space += 
+space +=
```
Comment (mfjarvis) [inline Makefile:153]: Dodgy merge conflict resolution?

Comment (booshibooshi) [reply Makefile:153]: Second space should be there and be equal to " ", it's a makefile workaround to remove a space from the username variable, cannot set soace variable to " " itself but you can append a space to an existing blank variable. Sure there's a better way of achieving this

Comment (mfjarvis) [reply Makefile:153]: Nope it's intellij being helpful

**Review by dan-jones-iw — APPROVED** [review]

**Review by dan-jones-iw — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (mfjarvis) [reply Makefile]: @alexmccormick-IW github didn't feel like showing me your message. I knew it was supposed to be there, the trouble was stopping intellij from removing it every time we commit :joy: Found another hacky way of defining a space instead

**Pull request merged**